src/pages/workspace/WorkspaceProfilePlanTypePage.tsx

```diff
@@ -49,6 +49,10 @@ function WorkspaceProfilePlanTypePage({policy}: WithPolicyProps) {
         OpenWorkspacePlanPage(policyID);
     }, [policyID]);
 
+    useEffect(() => {
+        setCurrentPlan(policy?.type);
+    }, [policy?.type]);
```

Comment:
We need to update the active selection UI too, it doesn't looks good upon update

<img width="382" alt="Screenshot 2024-12-24 at 3 43 45 PM" src="https://github.com/user-attachments/assets/81033293-1789-4cad-afaf-29134e62d273" />

Reply:
@jayeshmangwani

> We need to update the active selection UI too,

What do you mean? It is mentioned [here](https://github.com/Expensify/App/pull/54501#issuecomment-2560929327)

Reply:
Please ignore this; I forgot about [that](https://github.com/Expensify/App/pull/54501#issuecomment-2560929327).